src/vulkan-renderer/exceptions/vk_exception.cpp

```diff
@@ -148,16 +153,22 @@ namespace inexor::vulkan_renderer {
     case VK_ERROR_INVALID_DEVICE_ADDRESS_EXT:
         return "A buffer creation failed because the requested address is not available.";
     case VK_ERROR_FULL_SCREEN_EXCLUSIVE_MODE_LOST_EXT:
-        return "An operation on a swapchain created with VK_FULL_SCREEN_EXCLUSIVE_APPLICATION_CONTROLLED_EXT failed as "
-               "it did not have exlusive full-screen access. This may occur due to implementation-dependent reasons, "
+        return "An operation on a swapchain created with VK_FULL_SCREEN_EXCLUSIVE_APPLICATION_CONTROLLED_EXT "
+               "failed as "
+               "it did not have exlusive full-screen access. This may occur due to implementation-dependent "
+               "reasons, "
                "outside of  the application�s control.";
     case VK_ERROR_UNKNOWN:
-        return "An unknown error has occurred; either the application has provided invalid input, or an implementation "
+        return "An unknown error has occurred; either the application has provided invalid input, or an "
+               "implementation "
                "failure has occurred.";
     }
 
     // Don't forget to offer a default value.
     return "Unknown";
```

Comment:
Could be in the default case of the switch case. No idea what we use.

Reply:
Clang-tidy will complain without return.